src/extensions/export.ts

```diff
@@ -25,13 +24,16 @@ export function displayExportInfo(withRef: InstanceSpec[], withoutRef: InstanceS
   }
 }
 
-function displayParams(spec: InstanceSpec): void {
-  logger.info(humanReadable(spec));
-  logger.info(`Configuration will be written to 'extensions/${spec.instanceId}.env`);
-  for (const p of Object.entries(spec.params)) {
-    logger.info(`\t${p[0]}: ${p[1]}`);
+function displaySpecs(specs: InstanceSpec[]): void {
```

Comment:
Kind of hard to understand what this function is doing, can you add an example to the comment?